content-repo/extra-docs/articles/SystemDiagnosticsHealthCheck.md

```diff
@@ -0,0 +1,135 @@
+---
+id: system-diagnostics-and-health-check
+title: Integrations and Incidents Health Check
+description: The Health Check for Integrations and Incidents content pack uses out-of-the-box playbooks, scheduled as a job, to check for, return, and display information about failed integrations and incidents with errors. As part of the playbook run, users will be sent an email notification when failed incidents and/or integrations are discovered.
+---
+ 
+
+## What's in this Content Pack?
+ The XSOAR Health Check pack automatically reviewing the current server and content for issues and best practices. 
+The pack goal is to identify potential issues and remediate them before they become a real issue.
+
+This pack includes several content items that enable you to retrieve and view data about your system status and content configuration. It is automatically reviewing the current server and content for issues and suggest best practices procedures.
+The out-of-the-box items can easily customizable to fit your specific threasholds.
+- [Playbooks](#Playbooks)
+- [Incident Layouts](#Incident-Layouts)
+- [Dashboards & Lists](#Dashboards-and-Lists)
+
+
+### Playbooks
+This pack contains a parent playbook that calls two sub-playbooks.
+ 
+| Playbook | Description | Notes |
+|---------------- | ------------- | ------------- |
+| HealthCheck | This is the pack's main playbook and is the default playbook for the **System Diagnostics and Health Check** incident type. You should run this playbook as a manual incident. The playbook collects the data from all other subplaybooks and tasks and save it into XSOAR fields | Under the playbook inputs, you can add recipients to send the health check report via email. See below for additional features of this playbook. |
```

Comment:
Why is the playbook name is health check? We said before to change it with the System Diagnostics and Health Check.